app/scripts/directives/resources.js

```diff
@@ -194,8 +192,6 @@ angular.module('openshiftConsole')
         services: '=',
         portsByRoute: '=',
         showNodePorts: '=?',
-        // Optional empty message to display when there are no pods.
-        emptyMessage: '=?',
```

Comment:
Need to update `traffic-table.html`

https://github.com/spadgett/origin-web-console/blob/859a4cb2689c7bd8b5ef1a1c17ce8efcf267c758/app/views/directives/traffic-table.html#L15